internal/resources/aws/rds_cluster_instance.go

```diff
@@ -148,6 +108,86 @@ func (r *RDSClusterInstance) databaseEngineValue() string {
 	return "Aurora MySQL"
 }
 
+func (r *RDSClusterInstance) dbInstanceCostComponent(databaseEngine string) *schema.CostComponent {
+	purchaseOptionLabel := "on-demand"
+	priceFilter := &schema.PriceFilter{
+		PurchaseOption: strPtr("on_demand"),
+	}
+
+	var err error
+	if r.ReservedInstanceTerm != nil {
+		resolver := &rdsReservationResolver{
+			term:          strVal(r.ReservedInstanceTerm),
+			paymentOption: strVal(r.ReservedInstancePaymentOption),
+		}
+		priceFilter, err = resolver.PriceFilter()
+		if err != nil {
+			logging.Logger.Warn().Msgf(err.Error())
+		}
+		purchaseOptionLabel = "reserved"
+	}
+
+	// Example usage types for Aurora
+	// InstanceUsage:db.t3.medium
+	// InstanceUsageIOOptimized:db.t3.medium
+	// EU-InstanceUsage:db.t3.medium
+	// EU-InstanceUsageIOOptimized:db.t3.medium
+	usageTypeFilter := "/InstanceUsage:/"
+	if r.IOOptimized {
+		usageTypeFilter = "/InstanceUsageIOOptimized:/"
+	}
+
+	return &schema.CostComponent{
+		Name:           fmt.Sprintf("Database instance (%s, %s)", purchaseOptionLabel, r.InstanceClass),
+		Unit:           "hours",
+		UnitMultiplier: decimal.NewFromInt(1),
+		HourlyQuantity: decimalPtr(decimal.NewFromInt(1)),
+		ProductFilter: &schema.ProductFilter{
+			VendorName:    strPtr("aws"),
+			Region:        strPtr(r.Region),
+			Service:       strPtr("AmazonRDS"),
+			ProductFamily: strPtr("Database Instance"),
+			AttributeFilters: []*schema.AttributeFilter{
+				{Key: "instanceType", Value: strPtr(r.InstanceClass)},
+				{Key: "databaseEngine", Value: strPtr(databaseEngine)},
+				{Key: "usagetype", ValueRegex: strPtr(usageTypeFilter)},
+			},
+		},
+		PriceFilter: priceFilter,
+	}
+}
+
+func (r *RDSClusterInstance) auroraServerlessV2CostComponent(databaseEngine string) *schema.CostComponent {
+	var auroraCapacityUnits *decimal.Decimal
+	if r.CapacityUnitsPerHr != nil {
+		auroraCapacityUnits = decimalPtr(decimal.NewFromInt(*r.CapacityUnitsPerHr))
+	}
+
+	label := "Aurora serverless v2"
+	usageType := "Aurora:ServerlessV2Usage$"
+	if r.IOOptimized {
+		label = "Aurora serverless v2 (I/O-optimized)"
+		usageType = "Aurora:ServerlessV2IOOptimizedUsage$"
+	}
+
+	return &schema.CostComponent{
+		Name:           label,
+		Unit:           "ACU-hours",
+		UnitMultiplier: decimal.NewFromInt(1),
+		HourlyQuantity: auroraCapacityUnits,
+		ProductFilter: &schema.ProductFilter{
+			VendorName:    strPtr("aws"),
+			Region:        strPtr(r.Region),
+			Service:       strPtr("AmazonRDS"),
+			ProductFamily: strPtr("ServerlessV2"),
+			AttributeFilters: []*schema.AttributeFilter{
+				{Key: "databaseEngine", Value: strPtr(databaseEngine)},
+				{Key: "usagetype", ValueRegex: regexPtr(usageType)},
+			},
+		},
```

Comment:
Since this is a usage based cost component
```suggestion
		},
		UsageBased: true,
```